src/MultiUserEditingController.php

```diff
@@ -33,13 +33,7 @@ protected function init()
         //get the cache data
         $this->editingCache = Injector::inst()->get(CacheInterface::class . '.multiuserediting');
 
-
-        $usersEditing = unserialize($this->editingCache->get('editing'));
-
-        //create a new simple PHP object to store user editing data
-        if (!$usersEditing) {
-            $usersEditing = array();
-        }
+        $usersEditing = $this->editingCache->get('editing') ? unserialize($this->editingCache->get('editing')) : array();
```

Comment:
```suggestion
        $usersEditing = $this->editingCache->get('editing') ? unserialize($this->editingCache->get('editing')) : [];
```

Reply:
Yes. Sorry. I just copy and pasted the existing array() code. I have made this change now.